### PR TITLE
humans.txt

### PR DESCRIPTION
**Pull request by robsongajunior**
added humans.txt

### CODE REVIEW

File: humans.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* TEAM */
+
+	Chef: Jaydson Gomes
```
Comment (UltCombo) [inline humans.txt]: Chef = cooker.
"Leader", "Founder" or "Maintainer" would be more appropriate I guess.
cc @jaydson

File: humans.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	From: Aracajú
+
+	Theme developer: Robson Júnior
+	Contact: robson.ga.junior@gmail.com
+	Website: http://robsonjunior.cc
+	Twitter: @robsongajunior
+	From: Porto Alegre
+
+
+/* THANKS */
```
Comment (UltCombo) [inline humans.txt]: I believe the "Thanks" section is for people who contributed but are not really part of the team. Can you remove the "Thanks" section and move my data to below Jaydson?
Also, please add `Website: http://ultcombo.js.org`

File: humans.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	Maintainer: Fabrício Matté
+	Twitter: @Ult_Combo
+	From: Pelotas
+
+
+/* SITE */
+	Last update:2015/18/05
+	Language: Portuguese / English / Chinese
+	Doctype:HTML5
+	IDE: Sublime Text, Photoshop
```
Comment (UltCombo) [inline humans.txt]: add a line break at the end of the file so Git doesn't complain, please.

File: humans.txt (hunk 1 above)
Comment (jaydson) [inline humans.txt]: Maintainer

**Comment by UltCombo** [comment]
Nice work @robsongajunior, please take a look at the review comments when you have time.

**Comment by robsongajunior** [comment]
Aaaaaah nice, at night I will change, thanks for all support guys!

**Comment by UltCombo** [comment]
@robsongajunior ping

**Comment by robsongajunior** [comment]
@UltCombo, updated and break line added

**Comment by UltCombo** [comment]
Please move the `humans.txt` to inside the `resources` directory.
The rest looks fine, awesome work!

**Comment by robsongajunior** [comment]
> Please move the humans.txt to inside the resources directory.

@UltCombo

Done

**Comment by jaydson** [comment]
:clap:

**Pull request merged**